test/run-functional-tests

```diff
@@ -16,6 +16,7 @@ HA_NETWORK_V6_ARRAY[0]="2001:db8:10::/64"
 HA_NETWORK_V6_ARRAY[1]="2001:db8:20::/64"
 BEHAVE_CASE_DIR="$(dirname $0)/features/"
 BEHAVE_CASE_EXCLUDE="sbd|ocfs2"
+SUDOERS="/usr/etc/sudoers"
```

Comment:
Instead of modifying `/usr/etc/sudoers`, we should add configurations in `/etc/sudoers`.

From https://en.opensuse.org/openSUSE:Packaging_UsrEtc:

> All distribution-provided configuration files are located below /usr/etc and are not modifiable by the admin (with a read-only root filesystem, this would not even be possible)

Reply:
Changed